### ISSUE TITLE
Table editor feature requests

### ISSUE THREAD

**Issue by rickscr** [USEPA/SWMM-EPANET_User_Interface#45]
1. The table editors would be easier to use if they were rotated 90 degrees so the field names go horizontally and the records go vertically.  
2. Copy and paste from the tables only works for 1 cell at a time. It would be useful to have copy and paste enabled for multiple cells.

This:
![image](https://cloud.githubusercontent.com/assets/23215317/22700355/a0d7c62e-ed17-11e6-8ad2-5b6d0d7fdbe8.png)

Instead of this: 
![image](https://cloud.githubusercontent.com/assets/23215317/22700154/e593fa22-ed16-11e6-84b7-d5937ae72fe0.png)

**Comment by shenh** [comment]
I agree rotating 90 degrees would work better. It is logic to have properties as columns. 

Also it may take a long time to populate such a table with so many columns for large network. The rotated table may take less time to generate. 

Hailiang

**Comment by PaulDudaRESPEC** [comment]
Just implemented copy/paste using Ctrl-C and Ctrl-V for the table editor, resolving the second item.  For the first item, I'm going to suggest we put that one on hold since this version displays the properties the same as the former versions of the software, but it is a good suggestion for a future feature enhancement.
